### ISSUE TITLE
Unpredictable nested volume behavior in tether

### ISSUE THREAD

**Issue by jialin-li** [vmware/vic#5871]
As a developer, I hope to use nested volumes with predictable behavior. 

Right now volumes are stored in a map, and iterating through the volumes can give a different order each time. When starting a container with nested volumes, it may or may not succeed.

Tether debug log for this behavior is [here](https://gist.github.com/jialin-li/71bba93b1514ea4741ab6b0eaff7a454)
Step to reproduce (may need to run multiple times)
``` 
docker -H $HOST:2376 --tls create -i -v v1:/vol1 -v v2:/vol1/vol2 -v v3:/vol1/vol2/vol3 --name test busybox
docker -H $HOST:2376 --tls cp text.txt test:/vol1/vol2/vol3                                             
docker -H $HOST:2376 --tls start test

**Comment by jialin-li** [comment]
closed by #5865

**Comment by matthewavery** [comment]
This ticket was fixed by the PR which merged in docker-cp functionality(#5865)
